setupTests.js

```diff
@@ -0,0 +1,21 @@
+// JSDom + Vitest don't play well with each other. Long story short - default
+// TextEncoder produces Uint8Array objects that are _different_ from the global
+// Uint8Array objects, so some functions that compare their types explode.
+// https://github.com/vitest-dev/vitest/issues/4043#issuecomment-1905172846
```

Comment:
Does the following comment also fix the issue? It is a simpler workaround

https://github.com/vitest-dev/vitest/issues/4043#issuecomment-2577635283

Reply:
I could not get it to work with that solution.